chapters/testing.Rmd

```diff
@@ -572,31 +594,32 @@ might be a text file with a word frequency distribution that has a known \(\alph
 In order to create this text fixture,
 we need a way to generate random words.
 
-Fortunately, a Python library called `randomwordgenerator` exists to do just that.
-We can install it and the `pypandoc` library it depends on
-using [`pip`][pip], the Python Package Installer:
+Fortunately, a Python library called 
+[`RandomWordGenerator`](https://github.com/AbhishekSalian/Random-Word-Generator) 
+exists to do just that.
+We can install it using [`pip`][pip], the Python Package Installer:
 
 ```bash
-$ pip install pypandoc
-$ pip install randomwordgenerator
+$ pip install Random-Word-Generator
 ```
 
 Borrowing from the word count distribution we created for `test_alpha`,
-we can then create a text file full of random words
+we can use the following code to create a text file full of random words
 with a frequency distribution that corresponds to an \(\alpha\) of approximately 1.0:
 
 ```python
 import numpy as np
-from randomwordgenerator import randomwordgenerator as rwg
+from RandomWordGenerator import RandomWord
 
 
 max_freq = 600
 word_counts = np.floor(max_freq / np.arange(1, max_freq + 1))
-random_words = rwg.generate_random_words(n=max_freq)
+rw = RandomWord()
+random_words = rw.getList(num_of_words=max_freq)
 writer = open('test_data/random_words.txt', 'w')
 for index in range(max_freq):
     count = int(word_counts[index])
-    word_sequence = f"{random_words[index]} " * count
+    word_sequence = f'{random_words[index]} ' * count
     writer.write(word_sequence + '\n')
 writer.close()
 ```
```

Comment:
It might be worth showing the abbreviated output from 
```bash
$ cat test_data/random_words.txt
```
just to demonstrate that it is a file full of random words?

In #567 I've done the same earlier in this chapter to show that `word_counts` is a numpy array full of synthetic word count data.